corehq/apps/cloudcare/static/cloudcare/js/formplayer/main.js

```diff
@@ -3,6 +3,7 @@ hqDefine("cloudcare/js/formplayer/main", function () {
     $(function () {
         var initialPageData = hqImport("hqwebapp/js/initial_page_data").get;
         window.GMAPS_API_KEY = initialPageData('maps_api_key'); // maps api is loaded on-demand
```

Comment:
I don't think `maps_api_key` (the google maps api) is used anywhere else, I can remove this in this PR if we don't want to do a feature flag to toggle between googlemaps and mapbox, or I can get ride of it in a subsequent PR.

Reply:
👍